mvlearn/cluster/mv_kmeans.py

```diff
@@ -453,66 +532,36 @@ def fit(self, Xs):
             raise ValueError(msg)
 
         # Type and value checking for max_iter parameter
-        max_iter = np.inf
         if self.max_iter is not None:
             if not (isinstance(self.max_iter, int) and (self.max_iter > 0)):
                 msg = 'max_iter must be a positive integer'
                 raise ValueError(msg)
-            max_iter = self.max_iter
 
         # Type and value checking for n_init parameter
         if not (isinstance(self.n_init, int) and (self.n_init > 0)):
             msg = 'n_init must be a nonnegative integer'
             raise ValueError(msg)
 
+        # Type and value checking for tol parameter
+        if not (isinstance(self.tol, float) and (self.tol >= 0)):
+            msg = 'n_init must be a nonnegative integer'
```

Comment:
I think this should say "tol must be a nonnegative float"

Reply:
You're right